spec/system/admin/orders_spec.rb

```diff
@@ -277,6 +277,13 @@
 
         # non-empty cart order, with no with bill- and ship-address appear in the results
         expect(page).to have_content order_not_empty_no_address.number
+
+        # And the same orders are displayed when sorting by name:
+        find("th a", text: "NAME").click
+
+        expect(page).to have_no_content order_empty.number
```

Comment:
I thought that Capybara actually treated this the same way in more recent versions:

```suggestion
        expect(page).not_to have_content order_empty.number
```

But I can't immediately find proof of this. I think Rubocop should tell us which one to use..

Reply:
Yes, it's treated the same way. Do you have a preference? Reasons for or against?

Reply:
I think if there's no cause for a special matcher, then we should use the one way. It's slightly easier to see the `not` when it's in the expected position.

Of course, the difference is ever so slight.. 
I'll add it to my growing list of rules to suggest.